### PR TITLE
Natalia Lypovyk: Array of people

### PR DESCRIPTION
**Pull request by natalia-lypovyk**

### CODE REVIEW

File: src/arrayOfPeople.js [hunk 1]
```diff
@@ -38,7 +38,17 @@
  * @return {number[]}
  */
 function arrayOfPeople(people) {
-  // write code here
+  const peopleInRoom = [];
+
+  people.forEach(item => {
```
Comment (irenhh) [inline src/arrayOfPeople.js]:
```suggestion
  people.forEach(person => {
```

File: src/arrayOfPeople.js [hunk 2]
```diff
@@ -38,7 +38,17 @@
...
+    if (peopleInRoom.includes(item)) {
+      peopleInRoom.splice(peopleInRoom.indexOf(item), 1);
+    } else {
```
Comment (irenhh) [inline src/arrayOfPeople.js]: you can remove else

**Review by irenhh — CHANGES_REQUESTED** [review]

File: src/arrayOfPeople.js (hunk 2 above)
Comment (irenhh) [reply src/arrayOfPeople.js]: what about this comment?

**Review by irenhh — CHANGES_REQUESTED** [review]

File: src/arrayOfPeople.js (hunk 2 above)
Comment (natalia-lypovyk) [reply src/arrayOfPeople.js]: didn't notice it, sorry, changed

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+            "type": "node",
+            "request": "launch",
+            "name": "Launch Program",
+            "skipFiles": [
+                "<node_internals>/**"
+            ],
+            "program": "${workspaceFolder}\\src\\arrayOfPeople.js"
+        }
+    ]
+}
```
Comment (gestap) [inline .vscode/launch.json]: видали зайві файли

**Review by ost-ptk — APPROVED** [review]

**Pull request closed without merging**